backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/bpartner/JsonRequestConsolidateService.java

```diff
@@ -178,4 +189,29 @@ private void consolidateWithIdentifier(@NonNull final JsonRequestContactUpsertIt
 						.setParameter("jsonRequestContactUpsertItem", requestItem);
 		}
 	}
+
+	private static void consolidateWithIdentifier(@NonNull final JsonRequestBankAccountUpsertItem requestItem)
+	{
+		final ExternalIdentifier externalIdentifier = ExternalIdentifier.of(requestItem.getIdentifier());
+		switch (externalIdentifier.getType())
+		{
+			case METASFRESH_ID:
```

Comment:
shouldn't we cry here?

Reply:
nope, it's fine, here we just want to make sure any of the values that can be part of the identifier are consolidated with the identifier, that's how we do it for all the other objects (contacts, locations ...)